rule-object.libsonnet

```diff
@@ -0,0 +1,31 @@
+local mi = import 'mixin.libsonnet';
```

Comment:
can you create this file under extras folder?

Reply:
will do.